### ISSUE TITLE
Timestamps for single files

### ISSUE THREAD

**Issue by bb8** [aonez/Keka#261]
When using the single file mode (e.g. for GZIP), it'd be good to have an option to maintain the files' timestamps, much like the gzip cli utility.

**Comment by aonez** [comment]
@bb8 by default it maintains creation and modified times. Don't you mean those?

**Comment by bb8** [comment]
Yes, the creation and modified times. Using the GZIP method in Keka, with the option to archive as single files, the resulting .gz file has the date and time when the archive was created, instead of the original file's timestamp. This is in 1.1.3.

**Comment by aonez** [comment]
Ok so you're talking about the actual .gz file, not it's content.

**Comment by bb8** [comment]
Well both, because when extracting the .gz file with Finder and gunzip, the decompressed file appears to inherit the .gz file's creation and modification timestamps.

**Comment by aonez** [comment]
You're right. I'll investigate this one.

**Comment by aonez** [comment]
So files compressed individually (no tarball) are done using `p7zip`. Tarballs use `bsdtar` and `pigz` instead. Maybe using `pigz` directly will be a good idea.

**Comment by aonez** [comment]
The same applies to BZIP2 and XZ.
